modules/layers/src/bitmap-layer/bitmap-layer.js

```diff
@@ -108,6 +108,34 @@ export default class BitmapLayer extends Layer {
     }
   }
 
+  getPickingInfo({info}) {
+    const uniforms = this.state && this.state.model && this.state.model.uniforms;
+    const {bitmapTexture} = uniforms || {};
+
+    if (!info.color || !bitmapTexture) {
+      info.bitmap = null;
+      return info;
+    }
+
+    const {width, height} = bitmapTexture;
+
+    // Picking color doesn't represent object index in this layer
+    info.index = 0;
+
+    // Calculate uv and pixel in bitmap
+    const uv = unpackUVsFromRGB(info.color);
+    const pixel = [Math.floor(uv[0] * width), Math.floor(uv[1] * height)];
+
+    info.bitmap = {
+      size: {width, height}, // Size of bitmap
+      uv, // Floating point precision in 0-1 range
+      pixel, // Truncated to integer and scaled to pixel size
+      texture: bitmapTexture
```

Comment:
I would avoid storing the current texture in there. The picking info may be cached in multiple places and interfere with garbage collection.

Reply:
It is really useful to be able to get to it though, maybe a way to get it from the layer as discussed above (the pickInfo has a `layer` field).